Objects/regions.c

```diff
@@ -1507,7 +1552,11 @@ static int try_close(PyRegionObject *root_bridge) {
         // premeturely when the old data gets detached from it.
         regionmetadata_set_parent(old_data, NULL);
```

Comment:
Should `NULL` be the local region?

Reply:
As written on Teams, I prefer NULL to indicate that this clears the parent. But both are valid and behave the same.